packages/lesswrong/components/dropdowns/posts/HideFrontpagePostDropdownItem.tsx

```diff
@@ -31,11 +33,19 @@ const HideFrontpagePostDropdownItem = ({post}: {post: PostsBase}) => {
     ${fragmentTextForQuery("UsersCurrent")}
   `);
 
-  if (!currentUser || !allowHidingPosts) {
+  if (!allowHidingPosts) {
     return null;
   }
```

Comment:
EA Forum has wanted to make more actions visible (but still requiring login) to logged out users, and since this PR makes the three dots menu visible to logged out users I went ahead and also made the "hide from frontpage" and "subscribe to __" options visible to them as well.

This is what the post item three dots menu dropdown looks like for a logged out user on the EA Forum. I'm not really sure if we want to include the "subscribe to" options visible to logged out users - let me know if you think we shouldn't.

<img width="473" alt="Screen Shot 2023-06-16 at 9 35 19 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/dd2c9a07-4ab2-41ef-9f72-31464203f64c">